packages/core/src/components/scss/test.scss

```diff
@@ -0,0 +1,79 @@
+/* examples from https://sass-lang.com/guide/ */
+
+@use "sass:math";
+@use "partial.scss";
+@use 'button.scss' as Button;
+
+$font-stack: helvetica, sans-serif;
```

Comment:
Is this for test purposes?

Reply:
Yes, these are copied examples from scss home page for testing the features. I can change these to avoid confusion.

Reply:
Removed and added actual scss from hel.fi cookies